libff/algebra/scalar_multiplication/test_multiexp.cpp

```diff
@@ -18,7 +20,7 @@ void test_multiexp_accumulate_buckets(const size_t num_buckets)
     std::vector<bool> value_hit;
     value_hit.reserve(num_buckets);
     for (size_t i = 0; i < num_buckets; ++i) {
-        const bool hit = rand() % 2;
+        const bool hit = (i & 1) || rand() % 2;
```

Comment:
What's the rationale here behind the change `const bool hit = rand() % 2;` -> `const bool hit = (i & 1) || rand() % 2;`?

Reply:
The rationale is to ensure that we always have at least one value to be accumulated.

Reply:
Ok I see, thanks. What about dropping a comment above this line?

Reply:
Added a comment.